.github/workflows/lint.yml

```diff
@@ -5,6 +5,9 @@ on: [push, pull_request, workflow_dispatch]
 permissions:
   contents: read
 
+env:
+  RUFF_FORMAT: github
```

Comment:
Whilst we're here?
```suggestion
  FORCE_COLOR: 1
  RUFF_FORMAT: github
```

Reply:
pre-commit already enables colour for its logs, but we might as well have some nice colour for pip as well :)